### PR TITLE
Add method for pagination

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: UserBundle/Document/Group.php [hunk 1]
```diff
@@ -6,7 +6,10 @@
 use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
 
 /**
- * @ODM\Document(collection="group_document")
+ * @ODM\Document(
+ *  collection="group_document",
+ *  repositoryClass="OpenOrchestra\UserBundle\Repository\GroupRepository"
```
Comment (nicolasThal) [inline UserBundle/Document/Group.php]: There is an issue related on this, can you check it ?

File: UserBundle/Tests/Functional/Repository/UserRepositoryTest.php [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    public function testCountFilterSearch($descriptionEntity, $columns, $search, $count)
+    {
+        $users = $this->repository->countFilterSearch($descriptionEntity, $columns, $search);
+        $this->assertEquals($count, $users);
+    }
+
+    /**
+     * @return array
+     */
+    public function provideColumnsAndSearchAndCount(){
```
Comment (nicolasThal) [inline UserBundle/Tests/Functional/Repository/UserRepositoryTest.php]: psr

File: UserBundle/Tests/Functional/Repository/UserRepositoryTest.php [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    }
+
+    /**
+     * Generate columns of content with search value
+     *
+     * @param string $searchUsername
+     *
+     * @return array
+     */
+    protected function generateColumnsProvider($searchUsername = '')
```
Comment (nicolasThal) [inline UserBundle/Tests/Functional/Repository/UserRepositoryTest.php]: can you put this method just under the provider where you are using it, i think that would improve readability

Comment (itkg-alavieille) [inline UserBundle/Tests/Functional/Repository/UserRepositoryTest.php]: this method is used by several method, currently is just under the last provider wich use it.

Comment (nicolasThal) [inline UserBundle/Tests/Functional/Repository/UserRepositoryTest.php]: all right, i didn't saw it :)

File: UserBundle/Document/Group.php (hunk 1 above)
Comment (itkg-alavieille) [inline UserBundle/Document/Group.php]: Yes, it's this issue (https://github.com/open-orchestra/open-orchestra-cms-bundle/issues/774) but isn't this group document of groupBundle wich solve issue.

But this  PR https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/804 solve it

**Comment by itkg-alavieille** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**